### DISCUSSION TITLE
ux: better generated for long text

### DISCUSSION THREAD

**Discussion by rasvanjaya21** [47ng/nuqs#752] [Ideas] [answered]
**[term]**
The generated text breaks the ux so that the web page can be scrolled horizontally, this happens when the generated text is very long in one word (rare case such as looooooooooooooooooooooooooooooooooooooooong).

**[action]**
I add style `break-all` to handle this.

**[after]**
Before implement, generated text of l {o*168 } ng with strange ux.

![image](https://github.com/user-attachments/assets/d99e2454-52d2-4435-bd88-b08bce583153)

**[after]**
After implement, generated text of l {o*168 } ng with better ux.

![image](https://github.com/user-attachments/assets/45aaaa81-1771-4cdc-9d15-31be1f717d70)

**[request]**
I want to implement this feature 😁

**Comment by franky47** [comment, marked as answer]
Feel free to open a PR, thanks!
